frontend/public/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -111,16 +112,18 @@ export const OperatorHubItemDetails: React.SFC<OperatorHubItemDetailsProps> = ({
           <div className="co-catalog-page__overlay-body">
             <PropertiesSidePanel>
               { !installed
-                ? <Link
-                  className="btn btn-primary co-catalog-page__overlay-create"
-                  to={createLink}>
+                ? <Button
+                  className="co-catalog-page__overlay-create"
+                  component="a"
```

Comment:
This will reload the entire page. We need to use a `Link` component when navigating to another console URL.